### ISSUE TITLE
Another one that renders strangely

### ISSUE THREAD

**Issue by AndrewCRMartin** [JamesSweetJones/abYdraw#55]
This is a fusion antibody with the fusion protein tacked onto the C-terminus of each light chain (via a linker):

```
VH.a(1:13)-CH1(2:14){1}-H(3:9){2}-L(4)-CH2(5:11)-CH3(6:12)|VH.a(7:17)-CH1(8:18){1}-H(9:3){2}-L(10)-CH2(11:5)-CH3(12:6)|VL.a(13:1)-CL(14:2){1}-L(15)-X(16)|VL.a(17:7)-CL(18:8){1}-L(19)-X(20)
```

**Comment by JamesSweetJones** [comment]
<img width="1430" alt="Screenshot 2023-09-27 at 12 57 34" src="https://github.com/JamesSweetJones/abYdraw/assets/73530578/25ac673d-9bfb-42b8-932e-a78b47875d77">
Is this how it renders for you, @AndrewCRMartin?

**Comment by AndrewCRMartin** [comment]
No! The right hand `X` domain was flipped over and sitting on top of the right hand `CH1`.

Seems this must have been fixed and I'm not running the latest version...

**Comment by AndrewCRMartin** [comment]
Just cloned the latest version and I'm afraid it still isn't working for me!

![Screenshot from 2023-09-27 13-52-49](https://github.com/JamesSweetJones/abYdraw/assets/4394961/12f694dc-92b0-4e0e-a99d-f1218c1d02c5)

**Comment by JamesSweetJones** [comment]
Try with the latest version (1.09)
